src/test/java/io/cdap/plugin/ariba/source/metadata/proto/PropertiesTest.java

```diff
@@ -238,19 +242,32 @@ public void testGetNonObjectFields() {
   }
 
   @Test
-  public void teatValidateFields() throws AribaException, IOException {
+  public void testValidateFieldsWithoutAccessToken() {
+    MockFailureCollector collector = new MockFailureCollector();
+    AribaConnectorConfig connectorConfig = pluginConfig.getConnection();
+    connectorConfig.validateToken(collector);
+    Assert.assertEquals(1, collector.getValidationFailures().size());
+  }
+
+  @Test
+  public void testValidateFieldsWithInvalidResponse() throws AribaException, IOException {
     MockFailureCollector collector = new MockFailureCollector();
     AribaConnectorConfig connectorConfig = pluginConfig.getConnection();
+    aribaServices = new AribaServices(pluginConfig.getConnection());
+    new Expectations(AribaServices.class) {
+      {
+        aribaServices.getAccessToken();
+        result = "token";
+        minTimes = 0;
+      }
+    };
     connectorConfig.validateToken(collector);
-    collector.getOrThrowException();
-    Assert.assertEquals(0, collector.getValidationFailures().size());
+    Assert.assertEquals(1, collector.getValidationFailures().size());
   }
 
-  private void testTest(AribaConnector connector) {
+    private void testTest(AribaConnector connector) {

```

Comment:
fix indent

Reply:
Also, what is testTest()? Is it a testcase? If yes, annonate with @Test, make it public. Validate that there are no failures